### PR TITLE
760-fe-migrate-button-to-storybook

### PR DESCRIPTION
**Pull request by REALcuba**

### CODE REVIEW

File: packages/ui/src/components/atoms/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import { ButtonHTMLAttributes, FC } from 'react'
+import styled from 'styled-components'
+import { colors, dimensions, font } from '../../styles'
+
+const StyledButton = styled.button<TButton>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: ${dimensions.borderRadius.base};
+  // margin: ${dimensions.spacing.xxs};
```
Comment (kevinmamaqi) [inline packages/ui/src/components/atoms/Button.tsx:11]: Eliminar

File: packages/ui/src/components/atoms/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+  &:hover {
+    background-color: ${colors.primaryDark};
+    border: 2px solid ${colors.primaryDark};
+  }
+  &:active {
+    background-color: ${colors.primaryLight};
+    border: 2px solid ${colors.primaryLight};
+  }
+  &:disabled {
```
Comment (kevinmamaqi) [inline packages/ui/src/components/atoms/Button.tsx:32]: Añadir como condición

**Review by kevinmamaqi — CHANGES_REQUESTED** [review]
Migrar tests

File: packages/ui/src/components/atoms/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        secondary={secondary}
+        outline={outline}
+        size={size}
+        {...rest}
+    >
+        {children}
+    </StyledButton>
+)
+
+export default styled(Button)``
```
Comment (kevinmamaqi) [inline packages/ui/src/components/atoms/Button.tsx:97]: Eliminar y usar un named export

**Review by kevinmamaqi — CHANGES_REQUESTED** [review]

**Review by REALcuba — COMMENTED** [review]
done

**Review by REALcuba — COMMENTED** [review]
please review

File: packages/ui/src/__tests__/atoms/Button.test.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import React from 'react'
```
Comment (missvader) [inline packages/ui/src/__tests__/atoms/Button.test.tsx:1]: You don't need to import React, describe, expect or it, just only vi from 'vitest'.

File: packages/ui/src/__tests__/atoms/Button.test.tsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+import { fireEvent, render, screen } from '@testing-library/react'
+import { describe, expect, it, vi } from 'vitest'
+import { dimensions, colors } from '../../styles'
+import "@testing-library/jest-dom";
```
Comment (missvader) [inline packages/ui/src/__tests__/atoms/Button.test.tsx:5]: Unnecessary import, it´s already in setup files, Please remove it

File: packages/ui/src/components/atoms/Button.tsx [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+        data-testid="button"
+        secondary={secondary}
+        outline={outline}
+        size={size}
+        {...rest}
+    >
+        {children}
+    </StyledButton>
+)
+
```
Comment (missvader) [inline packages/ui/src/components/atoms/Button.tsx:94]: add import, export to index file

**Review by missvader — CHANGES_REQUESTED** [review]

File: packages/ui/src/__tests__/atoms/Button.test.tsx (hunk 4 above)
Comment (REALcuba) [reply packages/ui/src/__tests__/atoms/Button.test.tsx]: Done

File: packages/ui/src/__tests__/atoms/Button.test.tsx (hunk 5 above)
Comment (REALcuba) [reply packages/ui/src/__tests__/atoms/Button.test.tsx]: Done, thanks

File: packages/ui/src/components/atoms/Button.tsx (hunk 6 above)
Comment (REALcuba) [reply packages/ui/src/components/atoms/Button.tsx:94]: Done

**Review by kevinmamaqi — APPROVED** [review]

**Review by NataliaR-BCN — APPROVED** [review]

**Review by missvader — APPROVED** [review]

**Pull request merged**